src/current/v23.1/timestamp.md

```diff
@@ -66,8 +66,31 @@ CockroachDB supports precision levels from 0 (seconds) to 6 (microseconds) for `
 
 You can use an [`ALTER COLUMN ... SET DATA TYPE`]({% link {{ page.version.version }}/alter-table.md %}#alter-column) statement to change the precision level of a `TIMESTAMP`/`TIMESTAMPTZ`-typed column. If there is already a non-default precision level specified for the column, the precision level can only be changed to an equal or greater precision level. For an example, see [Create a table with a `TIMESTAMP`-typed column, with precision](#create-a-table-with-a-timestamp-typed-column-with-precision).
 
+## Supported casting and conversion
+
+`TIMESTAMP` values can be [cast]({% link {{ page.version.version }}/data-types.md %}#data-type-conversions-and-casts) to any of the following data types:
+
+Type | Details
+-----|--------
+`DECIMAL` | Converts to number of seconds since the Unix epoch (Jan. 1, 1970).
+`FLOAT` | Converts to number of seconds since the Unix epoch (Jan. 1, 1970).
+`TIME` | Converts to the time portion (HH:MM:SS) of the timestamp.
+`INT` | Converts to number of seconds since the Unix epoch (Jan. 1, 1970).
+`DATE` | --
+`STRING` |  Converts to the date and time portion (YYYY-MM-DD HH:MM:SS) of the timestamp and omits the time zone offset.
+
+### Infinity `TIMESTAMP` casts
+
+CockroachDB currently does not support an `infinity`/`-infinity` representation for `TIMESTAMP` casts. Instead, `infinity::TIMESTAMP` evaluates to `294276-12-31 23:59:59.999999+00:00`, the maximum `TIMESTAMP` value supported, and `-infinity::TIMESTAMP` evaluates to `-4713-11-24 00:00:00+00:00`, the minimum `TIMESTAMP` value supported.
+
+Note that this behavior differs from PostgreSQL, for which `infinity` is higher than any allowable `TIMESTAMP` value (including `294276-12-31 23:59:59.999999+00:00`), and `-infinity` is lower than any allowable `TIMESTAMP` value (including `-4713-11-24 00:00:00+00:00`).
+
+For more details, see [tracking issue](https://github.com/cockroachdb/cockroach/issues/41564).
```

Comment:
Ditto https://github.com/cockroachdb/docs/blob/main/StyleGuide.md#style-and-tone

```suggestion
```

Reply:
Removed - was confused for a sec b/c I didn't think this was a 'style and tone' thing but I found the section: https://github.com/cockroachdb/docs/blob/main/StyleGuide.md#github-issues-and-pull-requests